test/integration/configurations/nighthawk_http_origin_v2_api.yaml

```diff
@@ -0,0 +1,38 @@
+admin:
```

Comment:
Can we add a comment up here saying that this file is intentionally using v2 api because ... ?